src/index.ts

```diff
@@ -182,33 +182,35 @@ export abstract class BaseStorage {
     return changed
   }
 
-  protected rawGet = async (key: string): Promise<string> => {
-    if (this.hasExtensionApi) {
-      const dataMap = await this.#primaryClient.get(key)
+  protected rawGet = async (key: string): Promise<string | undefined> => {
+    return await this.rawGetMany([key])[key]
+  }
 
-      return dataMap[key]
+  protected rawGetMany = async (keys: string[]): Promise<Record<string, string | undefined>> => {
+    if (this.hasExtensionApi) {
+      return this.#primaryClient.get(keys)
```

Comment:
@sleekslush Does this work in Firefox?

Reply:
Yea, both chrome and firefox support arrays of keys